elks/include/linuxmt/biosparm.h

```diff
@@ -87,5 +87,8 @@ void BFPROC bios_copy_ddpt(void);
 struct drive_infot;
 void BFPROC bios_switch_device98(int target, unsigned int device,
         struct drive_infot *drivep);
+#ifdef CONFIG_BLK_DEV_BHD
+void bios_disk_park_all(void);
+#endif
```

Comment:
Let's get rid of ifdefs in header files to reduce build dependencies. There's no harm in leaving the forward declaration in, as the linker will always tell you a function is missing anyways.

Reply:
Done!